firmware-support/bittide-sys/src/time.rs

```diff
@@ -0,0 +1,321 @@
+// SPDX-FileCopyrightText: 2023 Google LLC
+//
+// SPDX-License-Identifier: Apache-2.0
+/*! Time structures.
+
+The `time` module contains structures used to get some sense of time.
+Keep in mind that in a Bittide system, clock frequencies are non-constant and thus there
+is no relation to wall clock time. However, we can use this module for a best effort
+approximation of wall clock time.
```

Comment:
While this is true, I'm not sure how much it matters. I think it needs a note that says that while clocks are dynamic in a Bittide system they're not expected to change more than a couple of ppm.

Reply:
Changed to:
```
While clocks are dynamic in a Bittide System, it is expected that the deviation from
wall clock time is no worse than any on-board static clock source and the effect is 
insignificant for timing purposes.
```